modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,131 @@
+
+* You have installed the latest version of the OpenShift CLI. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You have an {ibm-cloud-title} Platform account and ordered an {ibm-cloud-title} Object Storage plan and created an instance of {ibm-cloud-title} Object Storage. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
+
+.Procedure
+
+====
+
+. Fetch your credentials from the cluster deployed on {ibm-cloud-title}:
++
+[source,terminal]
+----
+export IBMCLOUD_API_KEY=$(oc get secrets ibmcloud-creds -n kube-system -o json | jq -r '.data.apiKey' | base64 -d)
+----
++
+. Log in to {ibm-cloud-title} by using {ibm-cloud-title} CLI:
++
+[source,terminal]
+----
+$ ibmcloud login -r us-east 
+----
++
+. Install the `cloud-object-storage` and `infrastructure-service` plugins:
++
+[source,terminal]
+----
+$ ibmcloud plugin install cloud-object-storage \
+  && \
+  ibmcloud plugin install infrastructure-service
+----
++
+. Create a resource group:
++
+[source,terminal]
+----
+$ ibmcloud resource group-create tempo-tracing \
+  && \
+  ibmcloud target -g tempo-tracing
+----
++
+. Create a service instance for the object store:
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create -d premium-global-deployment \
+  <tempo_bucket> cloud-object-storage standard global \
+  && \
+  export IBM_SERVICE_INSTANCE_ID=$(ibmcloud resource service-instance <tempo_bucket> -o json | jq -r '.[0].crn')
+----
++
+. Create an object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud cos bucket-create --bucket <tempo_bucket> --ibm-service-instance-id "$IBM_SERVICE_INSTANCE_ID"
+----
+====
+
+. Create a service key for connecting to the object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud resource service-key-create <tempo_bucket> Writer --instance-name <tempo_bucket> --parameters '{"HMAC":true}'
+----
+
+====
+
+. Create a secret in the {product-title} cluster with the {ibm-cloud-title} bucket credentials:
++
+[source,terminal]
+----
+$ oc <project> <secret_name>
+----
++
+. Extract the access and secret keys to be used on secret creation:
++
+[source,terminal]
+----
+$ json_output=$(ibmcloud resource service-key <tempo_bucket> -o json) \
+  && \
+  export IBM_BUCKET_ACCESS_KEY=$(echo $json_output | jq -r '.[0].credentials.cos_hmac_keys.access_key_id') \
+  && \
+  export IBM_BUCKET_SECRET_KEY=$(echo $json_output | jq -r '.[0].credentials.cos_hmac_keys.secret_access_key') \
+  && \
+  export IBM_BUCKET_ENDPOINT="https://s3.<region>.cloud-object-storage.appdomain.cloud" # <1>
+----
+<1> For example, `us-east`.
+
+====
+
+. Create a secret on {ibm-cloud-title}:
++
+[source,terminal]
+----
+$ oc -n tracing-system create secret generic ibm-cos-secret \
```

Comment:
@rubenvp8510, What's `tracing-system`? Is this a user-defined parameter? Is this the namespace?